helm_deploy/hmpps-education-and-work-plan-api/values.yaml

```diff
@@ -69,6 +69,10 @@ generic-service:
       HMPPS_SQS_QUEUES_EDUCATIONANDWORKPLAN_QUEUE_NAME: "sqs_queue_name"
     education-and-work-plan-domain-events-sqs-dl-instance-output:
       HMPPS_SQS_QUEUES_EDUCATIONANDWORKPLAN_DLQ_NAME: "sqs_queue_name"
+    education-and-work-plan-assessments-queue:
```

Comment:
I'm assuming the CPE stuff (new queue?) has been rolled out in preprod and prod? Otherwise we'll need to be careful about promoting other API changes forward into preprod and prod?

Reply:
yes it's in preprod and prod - much easier to do them all at once